src/components/header/header.css

```diff
@@ -1,14 +1,14 @@
 .header {
-  width: 100vw;
+  width: 100%;
   height: 4rem;
   display: flex;
+  gap: 32px;
   align-items: center;
   flex-direction: row;
   justify-content: space-between;
   background-color: #f0e21e;
 }
 .header-left,
-.header-center-right,
 .header-right {
   display: flex;
   align-items: center;
```

Comment:
En estas clases si agregaria un gap ya que el header tiene los elementos de cada contenedor muy juntos

<img width="2157" height="96" alt="Image" src="https://github.com/user-attachments/assets/8decec18-65c3-4516-b438-034723533bba" />


```suggestion
  align-items: center;
  gap: 32px;
```

<img width="2072" height="89" alt="Image" src="https://github.com/user-attachments/assets/66cf2d53-e8f5-494c-a39f-ef4c91fc75be" />